codecov.yml

```diff
@@ -1,2 +1,23 @@
+# Documentation: https://docs.codecov.io/docs/codecov-yaml
 ignore:
   - "api/v1alpha1/zz_generated.deepcopy.go" # generated file
+
+# general codecov settings
+codecov:
+    notify:
+        require_ci_to_pass: false
+coverage:
+    status:
+        # sets the expected status for `codecov/project` check.
+        # `target: auto` means it will use the current coverage to calculate against
+        #          the given PR/commit.
+        # theshold is the max delta allowed when compared with the current coverage
```

Comment:
`s/theshold/threshold/`. Also the line 12-13 description ends in a period and this one does not